CesiumGltf/src/PropertyTextureView.cpp

```diff
@@ -78,4 +44,71 @@ PropertyTextureView::getClassProperty(const std::string& propertyName) const {
 
   return &propertyIter->second;
 }
+
+PropertyTexturePropertyViewStatus PropertyTextureView::getTextureSafe(
+    const int32_t textureIndex,
+    int32_t& samplerIndex,
+    int32_t& imageIndex) const noexcept {
```

Comment:
This _might_ be a good use for `nonstd::expected`. So the signature of the function would be:

```cpp
nonstd::expected<std::reference_wrapper<Texture>, PropertyTexturePropertyViewStatus> getTextureSafe(const int32_t textureIndex) const noexcept;
```

The idea is that the function returns either a `Texture` reference (on success) or `PropertyTexturePropertyViewStatus` (on error) indicating the reason for failure.

That reference_wrapper is a bit unfortunate, though. It's needed because expected-lite apparently doesn't allow the success type to be a reference. It could be a `Texture*` instead, but it will never be null. I dunno, maybe this is too fancy and a bad idea, but I thought I'd mention it because using the Either monad for error handling is a cool technique if we don't use it here!